src/panels/lovelace/cards/hui-picture-entity-card.ts

```diff
@@ -0,0 +1,206 @@
+      (!config.image && !config.state_image && !config.camera_image)
+    ) {
+      throw new Error("No image source configured.");
+    }
+
+    this._config = { show_name: true, show_state: true, ...config };
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass) {
+      return html``;
+    }
+
+    const stateObj = this.hass!.states[this._config.entity];
+
+    if (!stateObj) {
+      throw new Error("State Ojbect does not exist for this entity");
+    }
+
+    const available = !(stateObj.state === UNAVAILABLE);
+
+    let footer = html``;
+    const name = this._config.name || computeStateName(stateObj);
+    const state = computeStateDisplay(
+      this.localize,
+      stateObj,
+      this.hass.language
+    );
+
+    if (this._config.show_name && this._config.show_state) {
+      footer = html`
+        <div class="footer both">
+          <div>${name}</div>
+          <div>${state}</div>
+        </div>
+      `;
+    } else if (this._config.show_name) {
+      footer = html`
+        <div class="footer">${name}</div>
+      `;
+    } else if (this._config.show_state) {
+      footer = html`
+        <div class="footer state">${state}</div>
+      `;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <ha-card
+        id="card"
+        @ha-click="${() => this.handleClick(false)}"
+        @ha-hold="${() => this.handleClick(true)}"
+        .longPress="${longPress()}"
+        class="${
+          classMap({
+            clickable: Boolean(available),
+          })
+        }"
+      >
+        <hui-image
+          .hass="${this.hass}"
+          .image="${this._config.image}"
+          .stateImage="${this._config.state_image}"
+          .cameraImage="${
+            computeDomain(this._config.entity) === "camera"
+              ? this._config.entity
+              : this._config.camera_image
+          }"
+          .entity="${this._config.entity}"
+          .aspectRatio="${this._config.aspect_ratio}"
+        ></hui-image>
+        ${footer}
+      </ha-card>
+    `;
+  }
+
+  private renderStyle(): TemplateResult {
+    return html`
+      <style>
+        ha-card {
+          min-height: 75px;
+          overflow: hidden;
+          position: relative;
+        }
+        hui-image.clickable {
+          cursor: pointer;
+        }
+        .footer {
+          @apply --paper-font-common-nowrap;
+          position: absolute;
+          left: 0;
+          right: 0;
+          bottom: 0;
+          background-color: rgba(0, 0, 0, 0.3);
+          padding: 16px;
+          font-size: 16px;
+          line-height: 16px;
+          color: white;
+        }
+        .both {
+          display: flex;
+          justify-content: space-between;
+        }
+        .state {
+          text-align: right;
+        }
+      </style>
+    `;
+  }
+
+  private handleClick(hold: boolean): void {
+    const stateObj = this.hass!.states[this._config!.entity];
```

Comment:
You should start the function with this:

```ts
const hass = this.hass!
const config = this._config!
```

Then you can get type checking back and drop all other `!`